src/scrapers/leumi.ts

```diff
@@ -200,6 +200,24 @@ async function navigateToLogin(page: Page): Promise<void> {
 }
 
 async function waitForPostLogin(page: Page): Promise<void> {
+  // Handle the new modal that appears after login
+  debug('Checking for login modal...');
+
+  try {
```

Comment:
Considre adding the logic as `waitForModalToClose` (or any other name) then here `await waitForModalToClose()`